src/cpp/xmlparser/XMLParserImpl.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef FASTRTPS_XMLPARSER_XMLPARSERIMPL_H_
+#define FASTRTPS_XMLPARSER_XMLPARSERIMPL_H_
+
+#include <fastrtps/xmlparser/XMLParser.h>
```

Comment:
```suggestion

#include <string>

#include <fastrtps/xmlparser/XMLParser.h>
```
Following the guideline "Include what you use".

Reply:
Done in e035ea991e1381ba57187339de12e6a3dc686370